csv_datasets/ldbc/lists/edges-sf-003-full-without-types.txt

```diff
@@ -14,51 +14,51 @@ sf-003/static/tag_hasType_tagclass/part-00000-0e79b266-fdeb-45c9-bc07-0df894854d
 sf-003/static/tagclass_isSubclassOf_tagclass/part-00000-3a05ed0e-440f-4389-bdd2-7ecabccdedc7-c000.csv.split_1
 
-creationDate|:START_ID(Comment)|:END_ID(Person)|:TYPE
+:IGNORE|:START_ID(Comment)|:END_ID(Person)|:TYPE
 sf-003/dynamic/comment_hasCreator_person/part-00000-6fabcea1-6ec8-4944-a25d-c8af3a1810fd-c000.csv.split_0
 sf-003/dynamic/comment_hasCreator_person/part-00000-6fabcea1-6ec8-4944-a25d-c8af3a1810fd-c000.csv.split_1
 
-creationDate|:START_ID(Comment)|:END_ID(Tag)|:TYPE
+:IGNORE|:START_ID(Comment)|:END_ID(Tag)|:TYPE
 sf-003/dynamic/comment_hasTag_tag/part-00000-17cded5e-b442-4075-97c9-51eb700574b5-c000.csv.split_0
 sf-003/dynamic/comment_hasTag_tag/part-00000-17cded5e-b442-4075-97c9-51eb700574b5-c000.csv.split_1
 
-creationDate|:START_ID(Comment)|:END_ID(Place)|:TYPE
+:IGNORE|:START_ID(Comment)|:END_ID(Place)|:TYPE
 sf-003/dynamic/comment_isLocatedIn_place/part-00000-b73c3da6-ab3b-4cc1-b5c1-39914cbb9f71-c000.csv.split_0
 sf-003/dynamic/comment_isLocatedIn_place/part-00000-b73c3da6-ab3b-4cc1-b5c1-39914cbb9f71-c000.csv.split_1
 
-creationDate|:START_ID(Comment)|:END_ID(Comment)|:TYPE
+:IGNORE|:START_ID(Comment)|:END_ID(Comment)|:TYPE
 sf-003/dynamic/comment_replyOf_comment/part-00000-72df4522-f48d-424c-9d01-11bfe0bc7cea-c000.csv.split_0
 sf-003/dynamic/comment_replyOf_comment/part-00000-72df4522-f48d-424c-9d01-11bfe0bc7cea-c000.csv.split_1
 
-creationDate|:START_ID(Comment)|:END_ID(Post)|:TYPE
+:IGNORE|:START_ID(Comment)|:END_ID(Post)|:TYPE
 sf-003/dynamic/comment_replyOf_post/part-00000-6bee34cb-0e59-454b-a96f-01bb8c448f42-c000.csv.split_0
 sf-003/dynamic/comment_replyOf_post/part-00000-6bee34cb-0e59-454b-a96f-01bb8c448f42-c000.csv.split_1
 
-creationDate|:START_ID(Forum)|:END_ID(Post)|:TYPE
+:IGNORE|:START_ID(Forum)|:END_ID(Post)|:TYPE
 sf-003/dynamic/forum_containerOf_post/part-00000-8e2eab71-bd45-48cb-b856-f62243cd1970-c000.csv.split_0
 sf-003/dynamic/forum_containerOf_post/part-00000-8e2eab71-bd45-48cb-b856-f62243cd1970-c000.csv.split_1
 
 creationDate|:START_ID(Forum)|:END_ID(Person)|:TYPE
 sf-003/dynamic/forum_hasMember_person/part-00000-808cfe18-6222-4e80-8eb7-6805821de22e-c000.csv.split_0
 sf-003/dynamic/forum_hasMember_person/part-00000-808cfe18-6222-4e80-8eb7-6805821de22e-c000.csv.split_1
 
-creationDate|:START_ID(Forum)|:END_ID(Person)|:TYPE
+:IGNORE|:START_ID(Forum)|:END_ID(Person)|:TYPE
 sf-003/dynamic/forum_hasModerator_person/part-00000-fb887ce0-a0d5-47c9-b013-ae8aa162462e-c000.csv.split_0
 sf-003/dynamic/forum_hasModerator_person/part-00000-fb887ce0-a0d5-47c9-b013-ae8aa162462e-c000.csv.split_1
 
-creationDate|:START_ID(Forum)|:END_ID(Tag)|:TYPE
+:IGNORE|:START_ID(Forum)|:END_ID(Tag)|:TYPE
 sf-003/dynamic/forum_hasTag_tag/part-00000-99e3e635-b013-4902-aad3-03c456b0d52c-c000.csv.split_0
 sf-003/dynamic/forum_hasTag_tag/part-00000-99e3e635-b013-4902-aad3-03c456b0d52c-c000.csv.split_1
 
-creationDate|:START_ID(Person)|:END_ID(Email)|:TYPE
+:IGNORE|:START_ID(Person)|:END_ID(Email)|:TYPE
```

Comment:
what is this one?

Reply:
@gliga this looks like one of those edges that don't exist in the actual schema but came out of the csv generator for some reason (idk why the generator would generate something not in the schema, @tylershunt / @danielmawhirter  might have more context)